cf/read_write/read.py

```diff
@@ -207,6 +210,10 @@ def read(
             be raised, unless the *ignore_read_error* parameter is
             True.
 
+            As a special case, if the `cdl_string` parameter is set to
+            True, the interpretation of `files` changes so that it
+            should be a string of CDL input rather than the above.
```

Comment:
To accommodate multiple strings, how about:
```
            As a special case, if the `cdl_string` parameter is set to
            True, the interpretation of `files` changes so that each value
            is assumed to be a string of CDL input rather than the above.
```